src/rt/sched/schedulerthread.h

```diff
@@ -898,8 +898,8 @@ namespace verona::rt
           {
             count++;
             *p = c->next;
-            c->dealloc(alloc);
             Systematic::cout() << "Stub collected: " << c << std::endl;
+            c->dealloc(alloc);
```

Comment:
ouch

Reply:
Was fine before, as it printed the pointer as a hex string, but not after this.  Glad we always run ASAN.

Reply:
as long as dealloc doens't null the pointer